src/android/device.rs

```diff
@@ -40,15 +108,72 @@ impl Reportable for ApkBuildError {
     }
 }
 
+#[derive(Debug)]
+pub enum AabBuildError {
+    BuildFailed(bossy::Error),
+}
+
+impl Reportable for AabBuildError {
+    fn report(&self) -> Report {
+        match self {
+            Self::BuildFailed(err) => Report::error("Failed to build AAB", err),
+        }
+    }
+}
+
+#[derive(Debug)]
+pub enum ApksBuildError {
+    CleanFailed(bossy::Error),
+    BuildFromAabFailed(bossy::Error),
+}
+
+impl Reportable for ApksBuildError {
+    fn report(&self) -> Report {
+        match self {
+            Self::CleanFailed(err) => Report::error("Failed to clean old APKS", err),
+            Self::BuildFromAabFailed(err) => Report::error("Failed to build APKS from AAB", err),
+        }
+    }
+}
+
+#[derive(Debug)]
+pub enum BundletoolInstallError {
```

Comment:
with how big the platform difference is, it'd probably be better to just have a macOS struct + non-macOS enum